src/lib/ngsi/ContextAttribute.h

```diff
@@ -87,10 +87,10 @@ typedef struct ContextAttribute
   std::string  toJsonAsValue(ConnectionInfo* ciP);
   void         present(const std::string& indent, int ix);
   void         release(void);
-  std::string  toString(void);
```

Comment:
I have take the opportunity of a name conflict to rename toString() and toStringValue() to usual getter names.

NTC